packages/zksend/src/experimental/wallet/index.ts

```diff
@@ -0,0 +1,454 @@
+		timestamp: new Date().toISOString(),
+	};
+};
+
+export class StashedWallet implements Wallet {
+	#events: Emitter<WalletEventsMap>;
+	#accounts: ReadonlyWalletAccount[];
+	#origin: string;
+	#name: string;
+
+	get name() {
+		return STASHED_WALLET_NAME;
+	}
+
+	get icon() {
+		return 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1NiIgaGVpZ2h0PSI1NiIgZmlsbD0ibm9uZSI+PHJlY3Qgd2lkdGg9IjU0IiBoZWlnaHQ9IjU0IiB4PSIxIiB5PSIxIiBmaWxsPSIjNTE5REU5IiByeD0iMjciLz48cmVjdCB3aWR0aD0iNTQiIGhlaWdodD0iNTQiIHg9IjEiIHk9IjEiIHN0cm9rZT0iIzAwMCIgc3Ryb2tlLXdpZHRoPSIyIiByeD0iMjciLz48cGF0aCBmaWxsPSIjMDAwIiBkPSJNMTguMzUzIDM1LjA2NGMuOTIxIDMuNDM4IDQuMzYzIDYuNTUxIDExLjQ4MyA0LjY0NCA2Ljc5NC0xLjgyMSAxMS4wNTItNy40MSA5Ljk0OC0xMS41My0uMzgxLTEuNDIzLTEuNTMtMi4zODctMy4zLTIuMjNsLTE1LjgzMiAxLjMyYy0uOTk3LjA3Ni0xLjQ1NC0uMDg4LTEuNzE4LS43MTYtLjI1Ni0uNTk5LS4xMS0xLjI0MSAxLjA5NC0xLjg1bDEyLjA0OC02LjE4M2MuOTI0LS40NyAxLjUzOS0uNjY2IDIuMTAxLS40NjguMzUyLjEyOC41ODQuNjM4LjM3MSAxLjI2N2wtLjc4MSAyLjMwNmMtLjk1OSAyLjgzIDEuMDk0IDMuNDg4IDIuMjUgMy4xNzggMS43NTEtLjQ2OSAyLjE2My0yLjEzNiAxLjU5OS00LjI0LTEuNDMtNS4zMzctNy4wOS02LjE3LTEyLjIyMy00Ljc5Ni01LjIyMiAxLjQtOS43NDggNS42My04LjM2NiAxMC43ODkuMzI1IDEuMjE1IDEuNDQ0IDIuMTg2IDIuNzQgMi4xNTdsMS45NzgtLjAwNWMuNDA3LS4wMS4yNi4wMjQgMS4wNDYtLjA0MS43ODQtLjA2NSAyLjg4LS4zMjMgMi44OC0uMzIzbDEwLjI4Ni0xLjE2NC4yNjUtLjAzOGMuNjAyLS4xMDMgMS4wNTYuMDUzIDEuNDQuNzE1LjU3Ni45OTEtLjMwMiAxLjczOC0xLjM1MiAyLjYzM2wtLjA4NS4wNzItOS4wNDEgNy43OTJjLTEuNTUgMS4zMzctMi42MzkuODM0LTMuMDItLjU4OWwtMS4zNS01LjA0Yy0uMzM0LTEuMjQ0LTEuNTUtMi4yMjEtMi45NzQtMS44NC0xLjc4LjQ3Ny0xLjkyNCAyLjU1LTEuNDg3IDQuMThaIi8+PC9zdmc+Cg==' as const;
+	}
+
+	get version() {
+		return '1.0.0' as const;
+	}
+
+	get chains() {
+		return [SUI_MAINNET_CHAIN] as const;
+	}
+
+	get accounts() {
+		return this.#accounts;
+	}
+
+	get features(): StandardConnectFeature &
+		StandardDisconnectFeature &
+		StandardEventsFeature &
+		SuiSignTransactionBlockFeature &
+		SuiSignTransactionFeature &
+		SuiSignPersonalMessageFeature &
+		SuiSignAndExecuteTransactionFeature {
+		return {
+			'standard:connect': {
+				version: '1.0.0',
+				connect: this.#connect,
+			},
+			'standard:disconnect': {
+				version: '1.0.0',
+				disconnect: this.#disconnect,
+			},
+			'standard:events': {
+				version: '1.0.0',
+				on: this.#on,
+			},
+			'sui:signTransactionBlock': {
+				version: '1.0.0',
+				signTransactionBlock: this.#signTransactionBlock,
+			},
+			'sui:signTransaction': {
+				version: '2.0.0',
+				signTransaction: this.#signTransaction,
+			},
+			'sui:signPersonalMessage': {
+				version: '1.1.0',
+				signPersonalMessage: this.#signPersonalMessage,
+			},
+			'sui:signAndExecuteTransaction': {
+				version: '2.0.0',
+				signAndExecuteTransaction: this.#signAndExecuteTransaction,
+			},
+		};
+	}
+
+	constructor({
+		name,
+		address,
+		origin = DEFAULT_STASHED_ORIGIN,
+	}: {
+		name: string;
+		network: StashedSupportedNetwork;
+		origin?: string;
+		address?: string | null;
```

Comment:
This can be removed since we don't support possession-of-address as an authentication model anymore